server.py

```diff
@@ -186,32 +173,11 @@ def show_user_tasks_and_gameplan():
 
     gameplan_tasks = GameplanTask.query.filter_by(user_id=user_id).order_by(GameplanTask.order).all()
```

Comment:
Seems unnecessary when all of this data is available on the `User` object. 
`user.settings`, `user.tasks` and `user.gameplantasks` should be all you need.